src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/appsettings.Development.json

```diff
@@ -19,5 +19,6 @@
   },
   "DataFiles": {
     "BasePath": "data/public-api-data"
-  }
+  },
```

Comment:
I guess with the fallback, you don't really need this value here.

Reply:
No that's true! I thought I'd leave it there for discoverability. But I'll remove it :)